### PR TITLE
SOC-993 | Special:UserLogout link to new login when new auth is enabled

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-993

### CODE REVIEW

**Comment by mklucsarits** [comment]
@kvas-damian This diff looks like it has a bunch of extra changes in it.

**Comment by kvas-damian** [comment]
Actually changes in messaging are required.
Initially link was generated from wikitext. Because we want to change URL conditionally I had to split message. That's why there are so many change.

File: languages/messages/MessagesHif_latn.php [hunk 1]
```diff
@@ -416,8 +416,9 @@
 # Login and logout pages
 'logouttext'                 => "'''Aap abhi logged out hai.'''
 
-Aap bina naam ke {{SITENAME}} ke kaam me lae sakta hai, nai to aap wahi sadasya ke naam se nai to duusra sadasya ke naam se [[Special:UserLogin|log in kare sakta hai]].
+Aap bina naam ke {{SITENAME}} ke kaam me lae sakta hai, nai to aap wahi sadasya ke naam se nai to duusra sadasya ke naam se  $1 .
```
Comment (mklucsarits) [inline languages/messages/MessagesHif_latn.php]: Looks like this string may not need a space before the period.

File: languages/messages/MessagesHsb.php [hunk 2]
```diff
@@ -545,8 +545,9 @@
 # Login and logout pages
 'logouttext'                 => "'''{{GENDER:|Sy|Sy}} nětko {{GENDER:|wotzjewjeny|wotzjewjena}}.'''
 
-Móžeš {{GRAMMAR:akuzatiw|{{SITENAME}}}} nětko anonymnje dale wužiwać abo so ze samsnym abo druhim wužiwarskim mjenom [[Special:UserLogin|zaso přizjewić]].
+Móžeš {{GRAMMAR:akuzatiw|{{SITENAME}}}} nětko anonymnje dale wužiwać abo so ze samsnym abo druhim wužiwarskim mjenom  $1 .
```
Comment (mklucsarits) [inline languages/messages/MessagesHsb.php]: Another space before the period.

File: languages/messages/MessagesIt.php [hunk 3]
```diff
@@ -653,8 +653,9 @@
 # Login and logout pages
 'logouttext'                 => "'''Logout effettuato.'''
 
-Si può continuare ad usare {{SITENAME}} come utente anonimo oppure [[Special:UserLogin|eseguire un nuovo accesso]], con lo stesso nome utente o un nome diverso.
+Si può continuare ad usare {{SITENAME}} come utente anonimo oppure  $1 , con lo stesso nome utente o un nome diverso.
```
Comment (mklucsarits) [inline languages/messages/MessagesIt.php]: Extra space before the comma.

File: languages/messages/MessagesKiu.php [hunk 4]
```diff
@@ -416,8 +416,9 @@
 # Login and logout pages
 'logouttext'               => "'''Sıma nıka cı ra veciyê.'''
 
-Sıma şikinê dızdêni {{SITENAME}} de dewam kerê, ya jê eyni karberi ya ki jê jüyê de bini [[Special:UserLogin|oncia cıkuyê]].
+Sıma şikinê dızdêni {{SITENAME}} de dewam kerê, ya jê eyni karberi ya ki jê jüyê de bini  $1 .
```
Comment (mklucsarits) [inline languages/messages/MessagesKiu.php]: Just check all of these. :)

**Comment by mklucsarits** [comment]
The code logic looks fine to me. Just make sure the spacing around the links are correct in the messages.

**Comment by kvas-damian** [comment]
Thanks @mklucsarits for catching it.
Fixed

**Comment by mklucsarits** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**